Examples/Examples.playground/Pages/01SwiftStdlibExtensions.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,141 @@
+//: [Table of Contents](00ToC)
+
+//: [Previous](@previous)
+
+import SwifterSwift
+
+//: ## SwiftStdlib extensions
+
+//: ### Array extensions
+
+// Remove duplicates from an array
+var array = ["h", "e", "l", "l", "o"]
+array.removeDuplicates()
+
+//: ### Bidirectional Collection extensions
```

Comment:
I'm not sure about the empty sections here. I think if its empty we can remove and add when we have examples for that :))